addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -1019,6 +1019,14 @@ var DocumentWidget = FileWidget.extend({
         }, options || {});
         this._super(parent, media, options);
     },
+    /**
+     * @override
+     */
+    async start() {
+        await this._super(...arguments);
+        // TODO Remove in master.
+        this.$addUrlButton.text(_t("Add URL"));
+    },
```

Comment:
What's this change for? The changed XML file in this PR is a static file, not an `ir.ui.view`, so existing DBs should receive the change already?